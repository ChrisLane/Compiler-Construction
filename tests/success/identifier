// This is to test identifier expresssions. This should parse.
main() {
    identifier = 10;
}
