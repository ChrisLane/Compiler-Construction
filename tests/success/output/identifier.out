main (  ) { 
    Asg ( Identifier "identifier" := Const 10 ) 
}
